README.md

```diff
@@ -1 +1,64 @@
-# book-store
+# Book store <img height="50" src="C:\Users\barma\IdeaProjects\book-store\images\logo small.png"/>
+
+The project was established with the goal to help book lovers to choose and to buy books online. 
+Inspired by Amazon of the early days. Book-store service is written in Java and this means that it has unlimited
+potential for further development. By now it has only basic functionality. 
+The selection of books is very large, but the interface is easy to understand and not overloaded with functionality,
+so you can quickly find the book you like.
+
+# Technologies been used <img height="50" src="C:\Users\barma\IdeaProjects\book-store\images\spring logo.png"/> <img height="50" src="C:\Users\barma\IdeaProjects\book-store\images\java 17 logo.png"/>
+Java 17 - as main language of the project.
+Spring Boot - as main framework to build this RESTful service 
+Spring Security - user as provider of user authentication, authorization, and role-based access control 
+Spring Data JPA - as source of a high-level solutions, easy-to-use interfaces for working with databases
+Liquibase - tool that maintains database schema version control. 
+Helps maintain consistency and automate database schema modifications across different environments and application releases.
+Maven - used to manage project dependencies.
+Swagger - as a way to visualize book-service through a user-friendly interface. By running an application and opening 
```

Comment:
use different .md construction to make your README nice-look

here you can use `list` to enumerate some points. 

```suggestion
- Java 17 - as main language of the project.
- Spring Boot - as main framework to build this RESTful service 
- Spring Security - user as provider of user authentication, authorization, and role-based access control 
- Spring Data JPA - as source of a high-level solutions, easy-to-use interfaces for working with databases
- Liquibase - tool that maintains database schema version control. 
Helps maintain consistency and automate database schema modifications across different environments and application releases.
- Maven - used to manage project dependencies.
- Swagger - as a way to visualize book-service through a user-friendly interface. By running an application and opening 
```